modules/ossm-cert-manager-verify.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+
+:_content-type: PROCEDURE
+[id="ossm-cert-manager-verify_{context}"]
+== Verify cert-manager installation
+
+To verify cert-manager is installed, follow these steps:
+
+. Check `istiod` certificate:
++
+[source, terminal]
+----
+kubectl exec $(kubectl get pods -l app=productpage -o jsonpath='{.items[0].metadata.name}') -c istio-proxy -- \
+    openssl s_client -showcerts istiod-basic.istio-system:15012 < /dev/null
```

Comment:
We should update these verification steps to validate against the root certificate. We should also specify the namespace in the command line to remove ambiguity.
```shell
kubectl exec "$(kubectl get pods -n bookinfo -l app=productpage -o jsonpath='{.items[0].metadata.name}')" -c istio-proxy -n bookinfo -- \
    openssl s_client -CAfile /var/run/secrets/istio/root-cert.pem -showcerts -connect istiod-basic.istio-system:15012 </dev/null
```